src/libcollections/btree/map.rs

```diff
@@ -1898,6 +1898,12 @@ impl<'a, K: Ord, V> VacantEntry<'a, K, V> {
         &self.key
     }
 
+    /// Take ownership of the key.
+    #[unstable(feature = "map_entry_keys", issue = "32281")]
```

Comment:
Oh right. Probably need a new feature as well, cause `map_entry_keys` is now stable. So just create a new issue "Tracking for map_entry_recover_keys" or some such?